frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts

```diff
@@ -316,11 +316,13 @@ export type ListPageHeaderProps = {
   badge?: React.ReactNode;
 };
 
+export type CreateAccessReviewProps = {
```

Comment:
The `Props` name here implies that these are top-level props to a component, which they aren't.